### PR TITLE
feat: isSlashCommand supports file parameters

### PR DESCRIPTION
**Pull request by xierenyuan**
支持参数是文件的场景 `/review src/component/hello.tsx `

### CODE REVIEW

File: src/slashCommand.test.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { describe, expect, test } from 'vitest';
+import { isSlashCommand } from './slashCommand';
+
+describe('isSlashCommand', () => {
```
Comment (sorrycc) [inline src/slashCommand.test.ts]: 1、用 test 组织就好，不用加 describe
2、用例精简下，很多 edge case 没必要
